fuzzers/forkserver_simple/src/main.rs

```diff
@@ -145,15 +145,16 @@ pub fn main() {
         None => [].to_vec(),
     };
 
+    let forkserver = ForkserverExecutorBuilder::new()
```

Comment:
Think I'd also add a wrapper called `ForkserverExecutor::builder` that calls through to this method. It's more consistent with other builders we use like launcher

Reply:
I need to set the types like this `<NopInput, (), (), StdShMemProvider>` to make it compile..
(otherwise it can't infer the types like OT, I, S)
```
impl ForkserverExecutor<NopInput, (), (), StdShMemProvider>
where
{
    /// Builder
    pub fn builder<'a>() -> ForkserverExecutorBuilder<'a, StdShMemProvider> {
        ForkserverExecutorBuilder::new()
    }
}
```
but is this correct...?
I feel the Input shoud not be constrained to NopInput